lib/galaxy/tool_util/xsd/galaxy.xsd

```diff
@@ -4412,6 +4412,30 @@ would appear as ``$options.vcf_output`` having the values ``'--vcf'`` when true
       </data>
     </outputs>
 ```
+
+Variables that have the type attribute ``select`` need a special python statement of
+the form ``selection and "option" in selection`` inside the ``<filter>`` tags. 
```

Comment:
```suggestion
Note that variables that correspond to optional select parameters are `None` if nothing is selected. Therefore a filter for such a variable looks like the following example.
```

Reply:
Maybe also describe what the variable looks like in case of `multiple="true"`

Reply:
Your "therefore" is quite strong.. the magic python statement still doesn't make much sense outside of this very special context...

Reply:
I think this is pretty standard. If you have a variable that is `None` if it is not set then you have to do this. One could be more explicit and write `outputs is not None and "save_period" in outputs` but this is equivalent. With the first statement you just assert that the outputs is set, i.e. it is a string, and the `in` operator works.

Reply:
yes you are right!